melosys-eessi-app/src/main/java/no/nav/melosys/eessi/integration/eux/rina_api/EuxConsumer.java

```diff
@@ -238,23 +250,24 @@ public void setSakSensitiv(String rinaSaksnummer) {
         log.info("Setter sak {} sensitiv", rinaSaksnummer);
 
         exchange(BUC_PATH + "/sensitivsak", HttpMethod.PUT, new HttpEntity<>(defaultHeaders()),
-                new ParameterizedTypeReference<Void>() {}, rinaSaksnummer);
+                new ParameterizedTypeReference<Void>() {
+                }, rinaSaksnummer);
```

Comment:
Kanskje heller prøve på en standard med hvert parameter på hver linje, istedenfor å breake linje mellom krøllparanteser for instansiering av en anonym klasse?

Syntes det så litt merkelig litt ut å breake her overalt 😬

Reply:
Jeg gjorde en cmd+alt+L for formattering, tydeligvis annet oppsett i Intellij